### ISSUE TITLE
error: could not find `Cargo.toml` when running `cargo install linera-sdk`

### ISSUE THREAD

**Issue by jasondavies** [linera-io/linera-protocol#1846]
Following the [installation instructions](https://linera.dev/getting_started/installation.html), I ran into this error pretty quickly when running `cargo install linera-sdk@0.9.0` on a fresh Ubuntu system:

```
$ cargo install linera-sdk@0.9.0
[... omitted for brevity ...]
error: failed to run custom build command for `linera-version v0.9.0`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installsuTUAf/release/build/linera-version-a6c643b74333d5ca/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /home/linera/.cargo/registry/src/index.crates.io-6f17d22bba15001f/linera-version-0.9.0/build.rs:26:63:
  called `Result::unwrap()` on an `Err` value: CargoMetadata(CargoMetadata { stderr: "error: could not find `Cargo.toml` in `/home/linera` or any parent directory\n" })
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `linera-sdk v0.9.0`, intermediate artifacts can be found at `/tmp/cargo-installsuTUAf`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**Comment by jasondavies** [comment]
Looks like this has been fixed by #1837.

**Comment by ma2bd** [comment]
Yes thanks! We are soon updating both the devnet and the dev manual to include the fix
